### PR TITLE
mongo overload script created

### PR DESCRIPTION
**Pull request by saareli123**
mongo overload script has been created
The script query simple find function and printing the request time to stdout

### CODE REVIEW

File: overload-test-scripts/mongo_overload.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+for t in threads:
+  t.join()
+
+
+# close the connection to the server
+client.close()
+
+
+# conn_str = 'mongodb://localhost:27017'
+# client = MongoClient(conn_str, serverSelectionTimeoutMS=5000)
```
Comment (Naor-Yekutiely) [inline overload-test-scripts/mongo_overload.py:47]: delete these lines

File: overload-test-scripts/mongo_overload.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  # execute the query and print the results
+  result = db.codegard_cache.find(query)
+  for r in result:
+    results.append(r)
+    # print(r)
+  elapsed_time = time.time() - start_time
+  print(f"Thread ID: {threading.get_ident()}, Elapsed time: {elapsed_time} seconds")
+
+# create a connection to the MongoDB server
+conn_str = 'mongodb://localhost:27017'
```
Comment (Naor-Yekutiely) [inline overload-test-scripts/mongo_overload.py:20]: Where this port is used?
In the docker-compose file?

Comment (saareli123) [reply overload-test-scripts/mongo_overload.py:20]: yes

File: overload-test-scripts/mongo_overload.py (hunk 1 above)
Comment (saareli123) [reply overload-test-scripts/mongo_overload.py:47]: OK

**Pull request merged**